oneview_redfish_toolkit/blueprints/computer_system_collection.py

```diff
@@ -48,8 +48,15 @@ def get_computer_system_collection():
             raise OneViewRedfishResourceNotFoundError(
```

Comment:
I know this behavior is previously than that current PR, but we should not raise exception when the list is empty, just return the empty list with status 200. 
We can do that right now. What do you think? @karolcatunda @victorhugorodrigues @adriane-cardozo @lhsvobodaj

Reply:
I agree.

Reply:
Yes, we should not raise exception for empty collections. Anyway, I think we should retrieve the list of server profiles directly instead of checking for each server hardware. Is there a reason why not doing that?

Reply:
Need to discuss how to handle profiles that are not assigned to a server hardware... maybe that's the reason :) Let's discuss tomorrow.